source/about/accessibile_html_error_messages.md

```diff
@@ -1,58 +1,58 @@
 
-Please note: These errors are only the result of our  inability to properly translate and display certain TeX and LaTex packages as HTML. These errors are not an indication of a larger system failure.
 
-## HTML errors readers may encounter
+## HTML errors you may see:
 
 When browsing and reading HTML papers on arXiv.org, you may see formatting errors in red like below: 
 
-![HTML LaTeX processing error for online id](images/reader-error-01.png)
+![HTML LaTeX processing error for online id](images/reader-error-01.png){.mkd-img-border}
 
-![HTML LaTeX processing error for tik Z library](images/reader-error-02.png)
+![HTML LaTeX processing error for tik Z library](images/reader-error-02.png){.mkd-img-border}
 
-![HTML LaTeX processing error for system name and revision](images/reader-error-03.png)
+![HTML LaTeX processing error for system name and revision](images/reader-error-03.png){.mkd-img-border}
 
 
-
-## HTML errors authors and submitters may encounter
-
-After you have submitted your paper and it has been processed, you will receive a link to preview the HTML version of your paper. You may see an error message alerting you that the TeX/LaTeX packages used to format your paper are not fully supported by our conversion process. Therefore your paper may display red TeX/LaTeX code and other formatting irregularities. 
-
 Please see an example of a HTML processing error below:
 ![html processing error](images/author_submssn_error.png){.mkd-img-full alt='Error message in a red box stating, This paper uses packages, listed below, that do not yet convert to HTML. These issues are known and are being worked on. View the list of unsupported packages.'}
 
 ## Frequently Asked Questions
 
-### What is Tex?
+### What is TeX?
 TeX is a text formatting and markup language first released in late 1970s by [D. Knuth](https://en.wikipedia.org/wiki/TeX#Hyphenation_and_justification). It was created to properly configure and visualize mathematical formulae, tables and symbols in printed documents such as journals and books. 
 
-### What is LaTex?
+### What is LaTeX?
 LaTeX is a TeX macro package developed in the early 1980s by [Leslie Lamport](https://en.wikipedia.org/wiki/LaTeX#cite_note-Lamport1986-4). It provides the ability to structure a document to meet different layout requirements such as section headers, footnotes, graphics and bibliographies.
 
-### Why are TeX and LaTex  so popular? 
-They are beloved by academic communities for their ability to easily display complex mathematical formulae, symbols, charts and graphs that common word processing programs have difficulty with. 
+### What is TeXML? 
+TeXML is an open source process that describes the markup elements of TeX using XML syntax. This allows XML data to be presented in PDF.
```

Comment:
do we use this tool? I am not aware of arXiv calling this tool, only latexml. Or does LaTeXML use this tool?

Reply:
I have updated the wording here and you will see it when I update this PR.